lib/traixroute/handler/handle_pch.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
Changing to just python has the risk that the user does not have python3 installed at all.

Reply:
pushed